src/DIRAC/WorkloadManagementSystem/Client/TornadoPilotLoggingClient.py

```diff
@@ -0,0 +1,29 @@
+from DIRAC.Core.Base.Client import Client, createClient
+
+
+@createClient("WorkloadManagement/TornadoPilotLogging")
+class TornadoPilotLoggingClient(Client):
+    def __init__(self, url=None, **kwargs):
+        """
+        Initialise a client.
+
+        :param str url: Server URL, if None, defaults to "WorkloadManagement/TornadoPilotLogging"
+        :param dict kwargs: additional keyword arguments, currently unused.
+        """
+        super(TornadoPilotLoggingClient, self).__init__(**kwargs)
+        if not url:
+            self.serverURL = "WorkloadManagement/TornadoPilotLogging"
+        else:
+            self.serverURL = url
+
+    def getMetadata(self):
```

Comment:
I don't think this is needed.

Reply:
What is not needed ? `getMetadata` ?

Reply:
yes, the method is not needed.

Reply:
It's because every `Client` has a sort of "passthrough" to the server-exposed functions (if look in the code they are all mostly "empty").